<a href="https://colab.research.google.com/github/Aline-C/hacking-civico/blob/master/02_python_101.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Tarea #2 📚
##**Parte 1: Densidad Poblacional**
---

### Cantidad de habitantes: 

- México:
119,938,473 personas

- Guanajuato:
5,864,777 personas

- León:
1,436,480 personas

### Superficie:

- México: 1 967 183 Km²

- Guanajuato: 30 589 Km²

- León: 1 220 Km²

> Información de habitantes obtenida del censo 2015 del [INEGI](https://www.inegi.org.mx/app/areasgeograficas/?ag=00)

> Datos de Extensión territorial obtenidos del [INAFED](http://www.inafed.gob.mx/work/enciclopedia/EMM11guanajuato/mediofisico.html)


In [1]:
datos = [
            {'Entidad':'México',
             'Población':119938473,
             'Área':1967183},
            {'Entidad':'Guanajuato',
             'Población': 5864777,
             'Área':30589},
            {'Entidad':'León',
             'Población': 1436480,
             'Área': 1220}
]

In [3]:
# Densidad poblacional = población / área ciudad

def densidad_poblacional (poblacion, area_ciudad):
  habitantes = poblacion ['Población']
  superficie = poblacion ['Área']

  densidad_p = habitantes // superficie
  return densidad_p

In [4]:
d = len(datos)
for index in range (d):
  dens_poblacional = densidad_poblacional(datos[index],datos[index])
  print ('Densidad poblacional de',datos[index]['Entidad'],':',dens_poblacional,'habitantes/km²')

Densidad poblacional de México : 60 habitantes/km²
Densidad poblacional de Guanajuato : 191 habitantes/km²
Densidad poblacional de León : 1177 habitantes/km²


### ¿Qué pasa al comparar varios estados o ciudades?

Nos podemos dar cuenta del contraste entre territorios. De como la cantidad de habitantes en las ciudades grandes es radicalmente diferente a la cantidad en ciudades pequeñas. Lo mismo entre estados.

#**Parte 2: Gasto per cápita**


---


### Presupuesto para *Sector Turismo*:
- Federal 2020: $5 034 491 561

- Federal 2019: $8 785 888 223

> Presupuesto de Egresos de la Federación para el Ejercicio Fiscal 2019 del [DOF](http://www.imss.gob.mx/sites/all/statics/pdf/leyes/4505.pdf)

> Presupuesto de Egresos de la Federación para el Ejercicio Fiscal 2020 [DOF](http://www.diputados.gob.mx/LeyesBiblio/pdf/PEF_2020_111219.pdf)

In [6]:
# Gasto per cápita = presupuesto / población

def gasto_p_capita (presupuesto,poblacion):
  total = presupuesto // poblacion
  return total

In [9]:
presupuesto_2020 = 5034491561
presupuesto_2019 = 8785888223
poblacion_mx = 119938473

gasto_p_capita_2020 = gasto_p_capita(presupuesto_2020,poblacion_mx)
print('Gasto per cápita para Sector Turismo 2020:', gasto_p_capita_2020,'MXN')

gasto_p_capita_2019 = gasto_p_capita (presupuesto_2019,poblacion_mx)
print ('Gasto per cápita para Sector Turismo 2019:', gasto_p_capita_2019,'MXN')

Gasto per cápita para Sector Turismo 2020: 41 MXN
Gasto per cápita para Sector Turismo 2019: 73 MXN


In [10]:
#porcentaje de cambio = gasto per cápita 2020 * 100 / gasto per capita 2019
porcentaje_de_cambio = gasto_p_capita_2020 * 100 // gasto_p_capita_2019
if porcentaje_de_cambio == 100:
  print('El presupuesto público fue igual al año anterior')
elif porcentaje_de_cambio > 100:
  print('El presupuesto público aumentó', porcentaje_de_cambio,'%')
else:
  print('El presupuesto público disminuyó', 100 - porcentaje_de_cambio,'%' )

El presupuesto público disminuyó 44 %


###¿En qué porcentaje aumentó o disminuyó?
Disminuyó un 44% con respecto al año 2019.

###¿A qué crees que se deba?
Me imagino que disminuyó en gran manera por el enfoque que el nuevo gobierno tiene: Austeridad.
Muchos de los egresos de gobierno se vieron disminuidos al no ser considerados de primera prioridad, o ser considerados como gastos prescindibles en el nuevo plan de gobierno.

De cierta manera, la disminución en este año al presupuesto de la Secretaría de Turismo puede verse como un acierto, puesto que la pandemia que vivimos actualmente ha impactado de gran manera a este sector.

#**Parte 3**


---


### ¿Para qué crees que nos pueda ser útil esta información?

Para tener herramientas de comparación y conscientización de como se administran los recursos del país y del enfoque que el gobierno esta tomando año con año. Esta información nos sirve como piedra inicial para empezar a cuestionarnos como las necesidades de las diferentes entidades van cambiando con el paso del tiempo y de qué estrategias se llevan a cabo para enfrentarlas.

###¿Qué decisiones podría tomar el gobierno basado en ésta?
Decisiones como el porcentaje de aumento o disminución de presupuestos para años posteriores dependiendo de la efectividad que tuvieron esos recursos a nivel federal, estatal y local. Teniendo los datos per cápita en los diferentes niveles de gobierno se puede comparar y evaluar la efectividad de las acciones de gobierno con los recursos que se tuvieron y tomar decisiones administrativas basándose en ello.